src/20240218_AaveV1Ethereum_AaveV1DeprecationPhase2/AaveV1Deprecation.md

```diff
@@ -18,29 +18,27 @@ While Aave v1 has been deprecated for a long time, some liquidity is still stuck
 
 This creates meaningful overhead for all involved development & security teams, which directly adds cost to the Aave DAO. Additionally, the architecture of Aave v1 is quite different to Aave v2 & v3, making it the most ad-hoc instance of Aave.
 
+On [6th of february](https://vote.onaave.com/proposal/?proposalId=15) a first set of measures was implemented by the DAO to accelerate the offboarding process.
+The proposed changes are in line with was suggested in the original offboarding plan for [phase 2](https://governance.aave.com/t/temp-check-bgd-further-aave-v1-deprecation-strategy/15893).
+
 ## Specification
 
 The proposal contract calls:
 
 - ADDRESSES_PROVIDER.setLendingPoolLiquidationManager(LIQUIDATION_MANAGER_IMPL);
-- ADDRESSES_PROVIDER.setLendingPoolImpl(POOL_IMPL);
-- CONFIGURATOR.setReserveInterestRateStrategyAddress(IR); on all assets where `IR` is a InterestRateStrategy with 0% base and 1%/2% slopes
 
 to update the implementations accordingly.
 
 This upgrade will:
 
-- reduce the interest rates to a minimal value to incentivize withdrawals
-- allow liquidations of collateralized positions with 1% liquidation bonus
-- allow liquidations with 100% close factor
-- disable flashloans
+- allow liquidations of collateralized positions with 3% liquidation bonus
```

Comment:
I would put also the new manager address etherscan link here

Reply:
done